yarn.lock

```diff
@@ -5,7 +5,6 @@
 "@automattic/calypso-build@5.0.1":
```

Comment:
lockfile changes look erroneous in this PR. I don't think there should be any, and you may be using an outdated version of yarn.

Reply:
Removed. i'm actually on a newer version I guess. I'm not really sure why the integrity checks are failing, but fairly convinced they don't matter.